src/input_xml.F90

```diff
@@ -975,6 +975,13 @@ subroutine read_geometry_xml()
       else
         call fatal_error("Must specify id of cell in geometry XML file.")
       end if
+
+      ! Copy cell name
+      c % name = ''
```

Comment:
It looks like all of the name components are initialized when the derived type is created. Is there a reason you set them to a blank string here as well? Seems superfluous unless I'm missing something.